components/errors/user.tsx

```diff
@@ -0,0 +1,21 @@
+import { ReactNode } from 'react';
+
+interface UserErrors {
+  children: ReactNode;
+}
+
+export default function UserErrors({ children }: UserErrors): JSX.Element {
+  return (
+    <div
+      className="govuk-error-summary optional-extra-class lbh-error-summary"
+      aria-labelledby="error-summary-title"
+      role="alert"
+      tabIndex={-1}
+      data-module="govuk-error-summary"
+    >
+      <h2 className="govuk-error-summary__title" id="error-summary-title">
+        {children}
+      </h2>
+    </div>
```

Comment:
Will likely need to check with design, but I think if we want to use this then we should ensure that it provides the necessary markup from [design system](https://design-system.hackney.gov.uk/components/error-summary). e.g.
```
<h2 className="govuk-error-summary__title" id="error-summary-title">{title}</h2>
<div class="govuk-error-summary__body">
    ...
</div>
```

Reply:
Reference in another Hackney repo:
https://github.com/LBHackney-IT/lbh-social-care-frontend/blob/main/components/ErrorSummary/ErrorSummary.tsx